privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/ui/PrivacyDashboardHybridActivity.kt

```diff
@@ -99,7 +99,7 @@ class PrivacyDashboardHybridActivity : DuckDuckGoActivity() {
                         is BrokenSiteForm -> {
                             when (params.reportFlow) {
                                 BrokenSiteForm.BrokenSiteFormReportFlow.MENU -> ReportFlow.MENU
-                                BrokenSiteForm.BrokenSiteFormReportFlow.PROMPT -> ReportFlow.PROMPT
+                                BrokenSiteForm.BrokenSiteFormReportFlow.PROMPT -> ReportFlow.RELOAD_THREE_TIMES_WITHIN_20_SECONDS
```

Comment:
Should we also rename `BrokenSiteForm.BrokenSiteFormReportFlow.PROMPT`?

Reply:
The only important thing is the string, but still, we should rename this for consistency. Done